classdefnotfounderror when running on hive-community-edition

```
2019-10-15 20:58:21,277 ERROR - Extension with id "hivemq-heartbeat-extension" cannot be started because of an uncaught exception thrown by the extension. Extension will be disabled.
java.lang.NoClassDefFoundError: javax/servlet/http/HttpServlet
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(Unknown Source)
	at java.base/java.security.SecureClassLoader.defineClass(Unknown Source)
	at java.base/java.net.URLClassLoader.defineClass(Unknown Source)
	at java.base/java.net.URLClassLoader$1.run(Unknown Source)
	at java.base/java.net.URLClassLoader$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.net.URLClassLoader.findClass(Unknown Source)
	at com.hivemq.extensions.classloader.IsolatedPluginClassloader.loadClass(IsolatedPluginClassloader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at com.hivemq.extensions.heartbeat.HeartbeatMain.startRestService(HeartbeatMain.java:76)
	at com.hivemq.extensions.heartbeat.HeartbeatMain.extensionStart(HeartbeatMain.java:59)
	at com.hivemq.extensions.HiveMQExtensionImpl.start(HiveMQExtensionImpl.java:133)
	at com.hivemq.extensions.HiveMQPlugins.pluginStart(HiveMQPlugins.java:209)
	at com.hivemq.extensions.loader.PluginLifecycleHandlerImpl.lambda$startPlugin$0(PluginLifecycleHandlerImpl.java:82)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ClassNotFoundException: javax.servlet.http.HttpServlet
	at java.base/java.net.URLClassLoader.findClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at com.hivemq.extensions.classloader.IsolatedPluginClassloader.loadClass(IsolatedPluginClassloader.java:123)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 18 common frames omitted
```

--- a/pom.xml
+++ b/pom.xml
@@ -21,7 +21,7 @@
     <name>HiveMQ Heartbeat Extension</name>
     <groupId>com.hivemq.extension</groupId>
     <artifactId>hivemq-heartbeat-extension</artifactId>
-    <version>1.0.0</version>
+    <version>1.0.1</version>
 
     <description>HiveMQ Heartbeat Extension</description>
 
@@ -122,7 +122,6 @@
                                     <exclude>com.hivemq:hivemq-extension-sdk</exclude>
                                     <exclude>org.slf4j:*</exclude>
                                     <exclude>ch.qos.logback:*</exclude>
-                                    <exclude>javax.servlet:*</exclude>
                                 </excludes>
                             </artifactSet>
                             <filters>

